src/pybind/mgr/volumes/fs/async_cloner.py

```diff
@@ -126,6 +126,25 @@ def sync_attrs(fs_handle, target_path, source_statx):
         log.warning("error synchronizing attrs for {0} ({1})".format(target_path, e))
         raise e
 
+def sync_xattrs(fs_handle, source_path, target_path):
+    try:
+        val = fs_handle.getxattr(source_path, "ceph.fscrypt.auth")
+    except cephfs.NoData:
+        return
+    except cephfs.Error as e:
+        log.warning(
+            "error synchronizing xattrs(getxattr) for {0} ({1})".format(target_path, e)
+        )
```

Comment:
nit: Using f-string can save you a couple of lines and also makes code more readable.
```suggestion
        log.warning(f"error synchronizing xattrs(getxattr) for {target_path} ({e})")
```